src/Lean/Parser/Command.lean

```diff
@@ -79,7 +80,7 @@ def structureTk          := leading_parser "structure "
 def classTk              := leading_parser "class "
 def «extends»            := leading_parser " extends " >> sepBy1 termParser ", "
 def «structure»          := leading_parser
-    (structureTk <|> classTk) >> declId >> many Term.bracketedBinder >> optional «extends» >> Term.optType
+    (structureTk <|> classTk) >> declId >> many (ppSpace >> Term.bracketedBinder) >> optional «extends» >> Term.optType
```

Comment:
See https://github.com/leanprover-community/mathlib3port/blob/9f61de08d4e62b7d0efe387e65b0af62f1a08d8a/Mathbin/Combinatorics/Quiver.lean#L47 for the problem being fixed here. (Note that all files in the mathlib3port repo are generated by `mathport`, and have come through the pretty printer.)

Reply:
Adding `ppSpace` here follows the example in `declSig`. (Thanks @digama0 for explaining this to me.)